Tests/Tests/UserInfoProviderSpec.swift

```diff
@@ -157,6 +157,106 @@ class UserInfoProviderSpec: QuickSpec {
                     expect(userInfoProviderA == userInfoProviderB).to(beTrue())
                 }
             }
+
+            context("when checking inequality") {
+                var userInfoProviderA: UserInfoProviderMock!
+                var userInfoProviderB: UserInfoProviderMock!
+
+                beforeEach {
+                    userInfoProviderA = UserInfoProviderMock()
+                    userInfoProviderB = UserInfoProviderMock()
+                }
+
+                it("will return true for nil object") {
```

Comment:
I think you meant "will return false..."

Reply:
Fixed here [e4ab56e](https://github.com/rakutentech/ios-inappmessaging/pull/257/commits/e9822a7e2d95d60d8837db3f031f2b990e4ab56e).